mod/videos.php

```diff
@@ -395,9 +395,9 @@ function videos_content(&$a) {
 	$o .= replace_macros($tpl, array(
 		'$title' => t('Recent Videos'),
 		'$can_post' => $can_post,
-		'$upload' => array(t('Upload New Videos'), $a->get_baseurl().'/videos/'.$a->data['user']['nickname'].'/upload'),
+		'$upload' => array(t('Upload New Videos'), App::get_baseurl().'/videos/'.$a->data['user']['nickname'].'/upload'),
 		'$videos' => $videos,
-        '$delete_url' => (($can_post)?$a->get_baseurl().'/videos/'.$a->data['user']['nickname']:False)
+        '$delete_url' => (($can_post)?App::get_baseurl().'/videos/'.$a->data['user']['nickname']:False)
```

Comment:
How about bringing `=>` and `=` aligned? I mean putting them with spaces under each other which makes the code (in my eyes) more readable.